pages/admin/lessons/[lessonSlug]/[pageName]/index.tsx

```diff
@@ -266,10 +360,14 @@ const Lessons = ({ data }: GetAppProps) => {
   const { pageName, lessonSlug } = router.query
 
   const { lessons } = data
-  const { data: modulesData, refetch } = useQuery<{ modules: Modules }>(
-    MODULES,
-    { fetchPolicy: 'no-cache' }
-  )
+  const { data: challengesData, refetch: refetchChallenges } =
+    useChallengesQuery({
+      fetchPolicy: 'no-cache'
+    })
+
+  const { data: modulesData, refetch: refetchModules } = useModulesQuery({
```

Comment:
To reduce the number of queries sent, we can only send a request to get x query (e.g, modules) when the user goes to its page. Let me know what you think.